packages/teams-js/src/public/mail.ts

```diff
@@ -4,6 +4,11 @@ import { FrameContexts } from './constants';
 import { runtime } from './runtime';
 
 export namespace mail {
+  /**
+   * Opens a mail message in Outlook.
```

Comment:
>Outlook [](http://example.com/codeflow?start=29&length=7)

This is only guaranteed to be true if Outlook is the host, right? Shouldn't this just be "Opens a mail message in the host?"